templates/show_params.mako

```diff
@@ -150,7 +150,16 @@
 </h2>
 
 <h3>Dataset Information</h3>
-<table class="tabletip">
+<table class="tabletip" id="dataset-details">
+    <thead>
+        <tr><th colspan="2" style="font-size: 120%;">
+            % if tool:
+                Tool: ${tool.name | h}
+            % else:
+                Unknown Tool
```

Comment:
This section showing the tool name was moved up into an h2 above, so this is now duplicated.